homepage/forms.py

```diff
@@ -0,0 +1,10 @@
+from django import forms
+
+
+class FilterForm(forms.Form):
+    choices = [('rate_over', 'rating over 3.5'), ('load_below', 'course load under 3.5'),
+               ('mand', 'mandatory'), ('elect', 'elective')]
+    filter_by = forms.MultipleChoiceField(
+        widget=forms.CheckboxSelectMultiple,
+        choices=choices,
+        required=False)
```

Comment:
it's more readable when the close parenthesis is in a new line
```suggestion
        required=False
   )
```